### PR TITLE
Add Err class for printing errors, update example file

### PR DESCRIPTION
**Pull request by AlecJStrickland**

### CODE REVIEW

**Comment by AlecJStrickland** [comment]
LGTM

**Comment by enterpriseloggingsolutions** [comment]
@AlecJStrickland - Hello contributor! The examples directory is not meant for perfectly working imports but instead samples of how this can work within your consuming applications. It obviously will never actually import from `java-logging` as this project is `java-logging`

File: example/example.ts [hunk 1]
```diff
@@ -1,4 +1,8 @@
+/* this file doesn't work, but here we are */
```
Comment (enterpriseloggingsolutions) [inline example/example.ts:1]: Remove this comment as this import is not expected to work

**Review by enterpriseloggingsolutions — CHANGES_REQUESTED** [review]

**Comment by AlecJStrickland** [comment]
Examples that are not meant to be directly executed are better left in a readme file. When one approaches this project and sees a file with a `.ts` extension, they are lead to believe that it in some way serves a function beyond being looked at.

File: example/example.ts (hunk 1 above)
Comment (AlecJStrickland) [reply example/example.ts:1]: I fully expected it to work

File: README.md [hunk 2]
```diff
@@ -7,8 +7,13 @@ We all know that Javascript is an insecure mess. The root of most of your applic
 ## Usage
 
 ```ts
+/* This is a fully working example */
```
Comment (enterpriseloggingsolutions) [inline README.md:10]: Please remove this comment

File: README.md [hunk 3]
```diff
@@ -7,8 +7,13 @@ We all know that Javascript is an insecure mess. The root of most of your applic
...
 import { System } from 'java-logging'
 
-System.out.println("very important business grade log") /* so much more maintinable */
-System.out.print("enterprise grade logging") /* far more clean */
+System.out.println("sup") /* so much more maintinable */
```
Comment (enterpriseloggingsolutions) [inline README.md:13]: these log messages do now showcase the enterprise grade quality of this logger. Please utilize the previous examples and come up with new ones that also show how scalable and portable this logging solution is

**Review by enterpriseloggingsolutions — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (AlecJStrickland) [reply README.md:12]: ok

File: README.md (hunk 2 above)
Comment (AlecJStrickland) [reply README.md]: ok

**Review by enterpriseloggingsolutions — APPROVED** [review]

**Pull request merged**